### PR TITLE
Docs: update readme

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

**Review by it09 — CHANGES_REQUESTED** [review]
It's a good start!
Some suggestions:
* keep benefits 
* move Current state of B to https://www.notion.so/pl-strflt/Compute-Over-Data-25fa366465ed4ddf81027498efb4a405
* move get the latest B updates to https://www.notion.so/pl-strflt/Compute-Over-Data-25fa366465ed4ddf81027498efb4a405
* checkout this blog on how to write README and what should be in it: https://github.com/18F/open-source-guide/blob/18f-pages/pages/making-readmes-readable.md

This is a list of README examples for reference
 - https://github.com/filecoin-project/mir
 - https://github.com/filecoin-project/neptune
 - https://github.com/filecoin-project/ref-fvm
 - https://github.com/filecoin-project/fvm-specs
 - https://github.com/filecoin-project/wasmtime - for a new project, you can add features/advantages section

**Comment by fakela** [comment]
> It's a good start! Some suggestions:
> 
> * keep benefits
> * move Current state of B to https://www.notion.so/pl-strflt/Compute-Over-Data-25fa366465ed4ddf81027498efb4a405
> * move get the latest B updates to https://www.notion.so/pl-strflt/Compute-Over-Data-25fa366465ed4ddf81027498efb4a405
> * checkout this blog on how to write README and what should be in it: https://github.com/18F/open-source-guide/blob/18f-pages/pages/making-readmes-readable.md
> 
> This is a list of README examples for reference
> 
> * https://github.com/filecoin-project/mir
> * https://github.com/filecoin-project/neptune
> * https://github.com/filecoin-project/ref-fvm
> * https://github.com/filecoin-project/fvm-specs
> * https://github.com/filecoin-project/wasmtime - for a new project, you can add features/advantages section

Thanks for the comment and suggestions @it09, just to be clear the current state of B and latest B updates, I should add the link to the notion instead than listing everything out.

**Comment by fakela** [comment]
@it09 I have made updates following the suggestions

File: README.md [hunk 1]
```diff
@@ -1,83 +1,120 @@
...
+- [Ways to contribute ](#ways-to-contribute)
+- [Current state of Bacalhau](current-state-of-bacalhau)
+- [License](#license)
 
-## Latest Updates
+## Features
+- **Process jobs fast**: Jobs in Bacalhau are processed where the data was created (meaning no ingress/egress) and all jobs are parallel by default.
+- **Low cost:** Bacalhau uses the compute that produced the data in the first place and reuses the existing hardware you already have. You also save on any ingress/egress fees you may have been charged. The cost of computing is much cheaper than managed services.
+- **Secure**: Data is not collected in a central location before processing, meaning all scrubbing and security can be applied at the point of collection.
+- **Large-scale data**: With Bacalhau, you can manage your own containers that can scale to batch process petabytes (quadrillion bytes) files.
```
Comment (simonwo) [inline README.md:62]: > you can manage your own containers

What management are we referring to? I’d rather be running processing without managing stuff!

**Review by simonwo — COMMENTED** [review]
Looks great!

**Comment by fakela** [comment]
@simonwo i have rephrased the sentences to be clear and just highlight what we offer

**Pull request closed without merging**